.github/workflows/ci.yml

```diff
@@ -20,8 +20,26 @@ jobs:
       options: --privileged
     steps:
     - uses: actions/checkout@v2
-    - name: Run storage tests
-      run: ./automation/tests-storage.sh
+    - name: Run storage tests not marked as root
+      run: |
+        ./automation/setup-storage.sh
+        chown -R vdsm:kvm ./ /var/tmp/vdsm*
+        su -c "./automation/tests-storage.sh" -- vdsm
+        make clean-storage
```

Comment:
This logic should move into automation/tests-storage.sh.

The github yaml should contain only code reuired to run on github. Anything else should be in automation/tests-storage.sh. This way if tomorrow you want to run the tests n gitlab (most tests already run on gitlab), we can change only the yaml file.

I think this can be something like:

    ./automation/tests-storage.sh vdsm